docs/api/app.md

```diff
@@ -1525,3 +1560,11 @@ or Windows [WOW](https://en.wikipedia.org/wiki/Windows_on_Windows)).
 You can use this property to prompt users to download the arm64 version of
 your application when they are running the x64 version under Rosetta
 incorrectly.
+
+### `app.notificationsComServerCLSID` _Windows_
+
+A `string` which contains fully formed CLSID for notifications COM server.
+This is necessary only for apps with persistent notifications support.
+The current default value for this property is `empty` (not `nil`). It means that by the default no persistent notifications support is exist.
+
+Changing for property value should be provided earlier than `ready` event triggered. Preferably changing for `app.notificationsComServerCLSID` is need to be provided into `will-finish-launching` event handler.
```

Comment:
As someone not too familiar with persistent notifications, it's unclear how these added APIs can be used together to add support for them.

We have a page in the docs for notifications which might be a good place to add an entry for persistent notifications.
https://www.electronjs.org/docs/latest/tutorial/notifications
https://github.com/electron/electron/blob/main/docs/tutorial/notifications.md

Maybe adding a brief explainer on what, why, and how to use persistent notifications on supported platforms would be helpful.

Reply:
Hello @samuelmaddock  if briefly Electron does not have support of persistent notifications at all. It means that there is any possibility to create toast with replay or permanent toast especially for MacOS, as I promised about month ago here is short promo with possibilities of newly API on MacOS. 
During tests I've used same app entry point as into my previous demonstration
https://github.com/electron/electron/files/8952901/app.zip

https://user-images.githubusercontent.com/89125090/190863358-d3af3ce0-0e49-40db-bb8e-6b414f1f58ed.mov
I think it is obviously that possibility of replies, buttons, handling clicks on Alert extremely exceeds that what we have now into electron. If briefly JS dev is not able to create real Alert and interact with it. 

Unfortunately since my last update I see new big stash regarding to non-persistent notifications which I have to adapt to have chance for push my changes. I will try to make mentioned changes ASAP but have my own tasks which are out of community.